src/App.tsx

```diff
@@ -48,47 +65,71 @@ export function getReorderedGoods(
 // };
 
 export const App: React.FC = () => {
+  const [isReversed, setReversed] = useState(false);
+  const [sortType, setSortType] = useState(SortType.NONE);
+
+  const reversed = () => {
+    setReversed(!isReversed);
```

Comment:
Better use previous state, to be clear that we are using the previous state, such a record will be more readable
```suggestion
    setReversed(prevState => !prevState);
```